### ISSUE TITLE
Bad English in https://nim-lang.org/docs/os.html

### ISSUE THREAD

**Issue by SoftwareApe** [nim-lang/Nim#12326]
Splitted is not the current spelling of the past participle of split.

### Current Output
The current example looks like this
```nim
import os

let myFile = "/path/to/my/file.nim"

let splittedPath = splitPath(myFile)
assert splittedPath.head == "/path/to/my"
assert splittedPath.tail == "file.nim"

assert parentDir(myFile) == "/path/to/my"

let splittedFile = splitFile(myFile)
assert splittedFile.dir == "/path/to/my"
assert splittedFile.name == "file"
assert splittedFile.ext == ".nim"

assert myFile.changeFileExt("c") == "/path/to/my/file.c"
```

### Expected Output
Since split is the correct form. It should be splitPath and splitFile, however I changed it to pathSplit and fileSplit before this causes naming collisions.

```nim
import os

let myFile = "/path/to/my/file.nim"

let pathSplit = splitPath(myFile)
assert pathSplit.head == "/path/to/my"
assert pathSplit.tail == "file.nim"

assert parentDir(myFile) == "/path/to/my"

let fileSplit = splitFile(myFile)
assert fileSplit.dir == "/path/to/my"
assert fileSplit.name == "file"
assert fileSplit.ext == ".nim"

assert myFile.changeFileExt("c") == "/path/to/my/file.c"
```

**Comment by hlaaftana** [comment]
This has been fixed in #12328, can close now.

**Comment by justingiffard** [comment]
Please mark as closed?
